src/api/dto/from/dtoToCriteria.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  ComparisonOperator,
+  CriteriaGroup,
+  CriteriaGroupType,
+  CriteriaTerminal,
+  CriteriaTerminalType,
+} from '../../../types/CriteriaTypes';
+import { FeeFineTypeDTO } from '../../queries/useFeeFineTypes';
+import { LocationDTO } from '../../queries/useLocations';
+import {
+  BursarExportFilterCondition,
+  BursarExportFilterDTO,
+  BursarExportFilterNegation,
+} from '../types';
+
+// inverse of ../to/criteriaToFilterDto
+export default function dtoToCriteria(
+  filter: BursarExportFilterDTO,
+  feeFineTypes: FeeFineTypeDTO[],
+  locations: LocationDTO[]
+): CriteriaGroup | CriteriaTerminal {
+  switch (filter.type) {
+    case 'Age':
```

Comment:
why not to avoid literals here?

Reply:
The `type` parameter is a known literal, as defined in https://github.com/folio-org/folio-export-common/blob/master/schemas/bursar-export/filter/bursarExportFilter.json

Reply:
but you already have the enum that is used in object, why not to use? by literal I mean `'Age'`, `'FeeFineOwner'` and any other magic words/numbers